### ISSUE TITLE
Should TrafficSignal require CoreDetails?

### ISSUE THREAD

**Issue by mdorford** [usdot-jpo-ode/wzdx#346]
The new FieldDeviceType, TrafficSignal, does not say that it requires CoreDetails.  Since all the other FieldDeviceType entries require CoreDetails, should this one?

**Comment by mark-mockett** [comment]
Yes, the [markdown file](https://github.com/usdot-jpo-ode/wzdx/blob/main/spec-content/objects/TrafficSignal.md) describing the TrafficSignal object indicates that it's required, but those requirements are missing from the [schema](https://github.com/usdot-jpo-ode/wzdx/blob/main/schemas/4.1/DeviceFeed.json). I'll open a PR to fix that - thanks for catching the issue!
